py-polars/tests/unit/operations/namespaces/test_binary.py

```diff
@@ -164,3 +166,65 @@ def test_binary_size(sz: int, unit: SizeUnit, expected: int | float) -> None:
         df["data"].bin.size(unit).item(),  # series
     ):
         assert sz == expected
+
+
+@pytest.mark.parametrize(
+    ("dtype", "type_size", "struct_type"),
+    [
+        (pl.Int8, 1, "b"),
+        (pl.UInt8, 1, "B"),
+        (pl.Int16, 2, "h"),
+        (pl.UInt16, 2, "H"),
+        (pl.Int32, 4, "i"),
+        (pl.UInt32, 4, "I"),
+        (pl.Int64, 8, "q"),
+        (pl.UInt64, 8, "Q"),
```

Comment:
Can we test `pl.Int128` as well?

Reply:
fixed.

Although, I should note that struct or numpy doesn't support ints. I created a second test function meant specifically for exotic ints (since it seems that UInt128 is WIP).
Reason to keep the old one, and not switch to this approach is the fact that `float` doesn't have `to_bytes` function.